nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Jc user add employee

# Description

I added a form that allows users to add employees.

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jc-user-add-employee`
4. go to employees, then hit the add employee button.
5. fill out the form and submit
6. the new employee should be added to the employee list

# Checklist:
